### PR TITLE
Unix timestamp format for DateTime fields in CSV

### PR DESCRIPTION
**Pull request by ludv1x**

### CODE REVIEW

File: dbms/src/IO/ReadHelpers.h [hunk 1]
```diff
@@ -767,7 +772,7 @@ inline void readDoubleQuoted(LocalDateTime & x, ReadBuffer & buf)
 
 /// CSV, for numbers, dates, datetimes: quotes are optional, no special escaping rules.
 template <typename T>
-inline void readCSVSimple(T & x, ReadBuffer & buf)
+inline void readCSVSimple(T & x, ReadBuffer & buf, void (*readText_)(T & x, ReadBuffer & buf) = readText)
```
Comment (alexey-milovidov) [inline dbms/src/IO/ReadHelpers.h]: Does this function get inlined and parameter eliminated?
Why not to use template parameter?

**Review by alexey-milovidov — CHANGES_REQUESTED** [review]

File: dbms/src/IO/ReadHelpers.h (hunk 1 above)
Comment (ludv1x) [inline dbms/src/IO/ReadHelpers.h]: Looks like that https://paste.yandex-team.ru/280275

Comment (ludv1x) [inline dbms/src/IO/ReadHelpers.h]: There is an other option:
```
readCSVSimple<decltype(x), readDateTimeText>(x, istr);
...
template <typename T, void (*readText_)(T & x, ReadBuffer & buf) = readText>
inline void readCSVSimple(T & x, ReadBuffer & buf) {...}
```

Comment (ludv1x) [inline dbms/src/IO/ReadHelpers.h]: I'm not sure what is more obvious.

**Pull request merged**